server/mdm/microsoft/microsoft_mdm.go

```diff
@@ -232,6 +238,12 @@ const (
 	// Login related query param expected by STS Auth endpoint
 	STSLoginHint = "login_hint"
 
+	// redirect_uri query param expected by TOC endpoint
+	TOCRedirectURI = "redirect_uri"
```

Comment:
Just double-checking that this is meant to refer to TOC and not TOS

Reply:
copypasta error. Good catch! fixed